index.html

```diff
@@ -1,544 +1,627 @@
+              <div class="control">
+                <button type="submit" class="button is-link">Submit</button>
+              </div>
+              <div class="control">
+                <button type="button" class="button is-text">Cancel</button>
+              </div>
+            </fieldset>
+          </form>
+        </div>
+      </section>
+
+      <section class="section">
+        <div class="container">
+          <div class="section-heading has-text-centered">
+            <h2 class="title is-2">Forum</h2>
+            <h3 class="subtitle is-5">Discuss any topic with no leaving this
+              page</h3>
+          </div>
+          <div class="columns">
+            <div class="column is-3">
+              <a href="/todo"
+                class="button is-primary is-block is-alt is-large">New Post</a>
             </div>
+            <div class="column is-9">
+              <div class="box content">
+                <article class="post">
+                  <h1>How do you center a button in a box?</h1>
+                  <div class="media">
+                    <div class="media-left">
+                      <p class="image is-32x32">
+                        <img
+                          src="https://source.unsplash.com/random/64x64/?sig=300"
+                          alt>
+                      </p>
+                    </div>
+                    <div class="media-content">
+                      <div class="content">
+                        <p>
+                          <a href="/todo" tabindex="-1">@jsmith</a> replied 34
+                          minutes ago &nbsp;
+                          <span class="tag">Question</span>
+                        </p>
+                      </div>
+                    </div>
+                    <div class="media-right">
+                      <span class="has-text-grey-light"><i
+                          class="fa fa-comments"></i> 1</span>
+                    </div>
+                  </div>
+                </article>
 
-            <div class="post">
-              <h4>How can I make a button go full width?</h4>
-              <div class="media">
-                <div class="media-left">
-                  <p class="image is-32x32">
-                    <img src="https://source.unsplash.com/random/64x64/?sig=301">
-                  </p>
-                </div>
-                <div class="media-content">
-                  <div class="content">
-                    <p>
-                      <a>@red</a> replied 40 minutes ago &nbsp;
-                      <span class="tag">Question</span>
-                    </p>
+                <article class="post">
+                  <h1>How can I make a button go full width?</h1>
+                  <div class="media">
+                    <div class="media-left">
+                      <p class="image is-32x32">
+                        <img
+                          src="https://source.unsplash.com/random/64x64/?sig=301"
```

Comment:
This image won't be read by a screenreader with alt missing. You can add something like `Red avatar`

Reply:
I'm not sure if the user needs to know that this is an avatar as far it's not interactive 🤔

Reply:
Every image even when it is not interactive should have alt attr. This attribute is fundamental for accessibility, ensuring that users who cannot see the images, such as those using screen readers, can still understand the content. 
https://d17btkcdsmqrmh.cloudfront.net/webaccessibility/docs/module-2/html/accessible_images